python/dolfinx/io/gmshio.py

```diff
@@ -258,6 +268,27 @@ def model_to_mesh(
         # Sort elements by ascending dimension
         perm_sort = np.argsort(cell_dimensions)
 
+        # Check that all cells are tagged once
+        _d = model.getDimension()
+        _elementTypes, _elementTags, _nodeTags = model.mesh.getElements(dim=_d, tag=-1)
+        # assert only one type of elements
```

Comment:
This looks like 1/2 code, 1/2 comment. Rephrase as a comment.

Reply:
Comment deleted